WcaOnRails/app/models/competition.rb

```diff
@@ -950,8 +973,9 @@ def has_location?
     latitude.present? && longitude.present?
   end
 
+  # The division is to convert the end result from secods to days. .to_date removed some hours from the subtraction
   def days_until
-    start_date ? (start_date - Time.now.utc.to_date).to_i : nil
+    start_date ? ((start_date - Time.now.utc)/(86_400)).to_i : nil
```

Comment:
To be perfectly precise here I believe we should do `start_date.to_time(:utc)`, otherwise `start_date` may be cast to a local `Time` 00:00, which is not what we want here.